pkg/vault/test/foundry/mutation/router/Router.t.sol

```diff
@@ -113,10 +155,45 @@ contract RouterMutationTest is BaseVaultTest {
         router.swapSingleTokenHook(params);
     }
 
+    function testSwapSingleTokenHookReentrancy() public {
+        vm.expectRevert(ReentrancyGuardTransient.ReentrancyGuardReentrantCall.selector);
+        router.manualReentrancySwapSingleTokenHook();
+    }
+
+    /*
+        addLiquidityToBufferHook
+            [x] onlyVault
+            [x] nonReentrant
+    */
```

Comment:
```suggestion
```
Maybe just get rid of these Pep headers everywhere? They don't match the style of the rest of the codebase, and the test names are descriptive enough.